### PR TITLE
Add make_release and next_release scripts

### PR DESCRIPTION
**Pull request by alixdamman**
Not tested yet.

### CODE REVIEW

**Comment by alixdamman** [comment]
Only tested `add_release` function (`next_release.py` module) until call to `push` on Linux and Windows. 
But ready for a first round of review of `main.py`, `config.py`, `utils.py` and `next_release.py` modules

**Comment by alixdamman** [comment]
@gdementen Do I move the content of `check_local_repo` to `check_clone`?

File: releaser/config.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+# coding=utf-8
+from __future__ import print_function, unicode_literals
+
+import sys
+from os.path import join
+from releaser.utils import git_remote_last_rev
+
+TMP_DIR = r"c:\tmp" if sys.platform == "win32" else "/tmp"
+
+LARRAY_CONFIG = {'project_name': 'liam2',
```
Comment (gdementen) [inline releaser/config.py]: the key "project_name" is a bit ambiguous. => change it to e.g. "github_user"?

File: releaser/config.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                'announce_group': 'liam2-announce@googlegroups.com',
+                'users_group': 'liam2-users@googlegroups.com',
+                }
+
+CONFIG = {'larray': LARRAY_CONFIG, 'larray-editor': EDITOR_CONFIG, 'larray-eurostat': EUROSTAT_CONFIG,
+          'liam2': LIAM2_CONFIG}
+
+
+def get_config(package_name, release_name, branch):
+    package_config =  CONFIG[package_name]
```
Comment (gdementen) [inline releaser/config.py]: kill extra space

File: releaser/config.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    module_name = package_config['module_name']
+
+    repository = "https://github.com/{}/{}".format(project_name, package_name)
+    rev = git_remote_last_rev(repository, 'refs/heads/{}'.format(branch))
+    public_release = release_name != 'dev'
+    if not public_release:
+        # take first 7 digits of commit hash
+        release_name = rev[:7]
+
+    config = {'package_name': package_name,
```
Comment (gdementen) [inline releaser/config.py]:
```python
config = package_config.copy()
config.update({
    ....
})
```

**Review by gdementen — COMMENTED** [review]
A big config dict for all projects is not how I would have done it. I would have kept an independent script in each project. Those scripts would only be a few lines long: define a set of constants and call a function defined in the "releaser" module (e.g. make_release). This seems better in the long run. This can be changed later though.

**Comment by gdementen** [comment]
> @gdementen Do I move the content of check_local_repo to check_clone?

No! Those are two very different functions which do not even act on the same repositories!

File: releaser/config.py (hunk 1 above)
Comment (alixdamman) [reply releaser/config.py]: github_project?

Comment (gdementen) [reply releaser/config.py]: No, that is not how you use it. The usual vocabulary is user and repo(sitory), as for example in: https://github.com/user/repo

See e.g. https://help.github.com/articles/adding-a-remote/

**Review by gdementen — APPROVED** [review]

**Pull request merged**